### ISSUE TITLE
not showing miners.

### ISSUE THREAD

**Issue by karkaletsi** [UnamSanctam/UnamWebPanel#125]
![image](https://user-images.githubusercontent.com/104867512/166585310-42b73e78-3cb3-4e56-bbdf-30db973e507f.png)
not showing miners.

Added the api in miner

MYIP/api/endpoint.php

Checked if i added https instead of http but everything good.

**Comment by UnamSanctam** [comment]
Where are you hosting it (since you said MYIP)?

**Comment by karkaletsi** [comment]
[Private-hosting](https://private-hosting.eu/) i hosted it before and worked fine.. few months later doesnt work

**Comment by UnamSanctam** [comment]
Check if you're able to create a new configuration in the "Configuration" tab, you can just enter any name and then {} as the configuration. Also check the `__UNAM_LIB\Logs` folder for any error files, you can also check the `api` folder. And I'm guessing you don't have a certificate (since you said you use http)?

**Comment by karkaletsi** [comment]
no certificate. Last time i had https . was that the problem?

**Comment by karkaletsi** [comment]
cant create config

**Comment by UnamSanctam** [comment]
If you can't create a configuration then most likely either you don't have write permissions to the database or your PHP version is too old (should be PHP 7.4 or above).

**Comment by karkaletsi** [comment]
php 7.4.3 is the version i have

**Comment by karkaletsi** [comment]
database it is.. idk tho i can connect fine.

**Comment by UnamSanctam** [comment]
Yes, but it most likely can't write anything to the database so it can't add miners.

**Comment by karkaletsi** [comment]
reinstalled database
![image](https://user-images.githubusercontent.com/104867512/166605124-ba945087-4e3c-4c8d-84a4-0f984a90fccd.png)

i tried now to add new config still nothing

**Comment by UnamSanctam** [comment]
You most likely don't have write permissions to the unamwebpanel.db file (the SQLite database).

**Comment by karkaletsi** [comment]
cd /var/www/html/ 
chmod +x unamwebpanel.db?

**Comment by karkaletsi** [comment]
i added chmod 777 to the whole folder

**Comment by UnamSanctam** [comment]
As long as your webserver user has write permissions to the unamwebpanel.db (and its parent folder) it will be able to write.
